README.md

```diff
@@ -1154,6 +1156,7 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Icecast 2](https://icecast.org) - streaming audio/video server which can be used to create an Internet radio station or a privately running jukebox and many things in between. ([Source Code](https://gitlab.xiph.org/xiph/icecast-server), [Clients](https://icecast.org/apps/)) `GPL-2.0` `C`
 - [Jellyfin](https://jellyfin.org) - Media server for audio, video, books, comics, and photos with a sleek interface and robust transcoding capabilities. Almost all modern platforms have clients, including Roku, Android TV, iOS, and Kodi. ([Demo](https://demo.jellyfin.org/stable), [Source Code](https://github.com/jellyfin/jellyfin)) `GPL-2.0` `C#`
 - [Karaoke Forever](https://www.karaoke-forever.com) - Host awesome karaoke parties where everyone can easily find and queue songs from their phone's web browser. The player is also browser-based with support for MP3+G, MP4 and WebGL visualizations. ([Source Code](https://www.karaoke-forever.com/repo)) `ISC` `Nodejs`
+- [Kodi](https://kodi.tv/) - Multimedia/Entertainment center, formerly known as XBMC. Runs on Android, BSD, Linux, macOS, iOS and Windows. ([Source Code](https://github.com/xbmc/xbmc)) `GPL-2.0` `C++`
```

Comment:
Kodi is great, but I'm not sure it counts as self-hosted? As far as I know it doesn't have a remotely accessible interface, it's a purely local media player. If I'm correct please remove it from this PR.

Reply:
Kodi has [a web interface](https://kodi.wiki/view/Web_interface) that can be configured and used to stream content to other devices. See also [[1]](https://www.makeuseof.com/tag/how-to-use-kodi-web-interface/) [[2]](https://www.addictivetips.com/media-streaming/kodi/web-interface-access-files/). Is it enough to count it as self-hosted?

Reply:
Ping @nodiscc

Reply:
> Kodi has [a web interface](https://kodi.wiki/view/Web_interface) that can be configured and used to stream content to other devices

Ok :+1: